src/components/Expenses.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 
 import Input from './Input';
+import Button from './Button';
 
 import './dashboard.css';
```

Comment:
This should not be using the dashboard styles, it should have it's own stylesheet.